Jenkinsfile

```diff
@@ -60,60 +88,60 @@ timestamps() {
 					sh 'npx retire --exitwith 0'
 
 					step([$class: 'WarningsPublisher', canComputeNew: false, canResolveRelativePaths: false, consoleParsers: [[parserName: 'Node Security Project Vulnerabilities'], [parserName: 'RetireJS']], defaultEncoding: '', excludePattern: '', healthy: '', includePattern: '', messagesPattern: '', unHealthy: ''])
-				} // stage
+				} // nodejs
+			} // stage
+		} // node
```

Comment:
again, clean up the workspace on the node before you return.

Reply:
Added this [here](https://github.com/ewanharris/alloy/blob/966becacd88eb2a41985571766dd8ba3cfc3a232/Jenkinsfile#L96)